src/olympia/reviewers/tests/test_serializers.py

```diff
@@ -243,6 +243,26 @@ def test_uses_unknown_minified_code(self):
         data = self.serialize(fobj, file='background-script.js')
         assert not data['uses_unknown_minified_code']
 
+    def test_can_exclude_entries(self):
```

Comment:
The names of the different test cases make me think that it would be a good idea to rename the `file_only` variable to `exclude_entries`.